userspace/libsinsp/sinsp.cpp

```diff
@@ -1524,7 +1516,6 @@ int32_t sinsp::next(OUT sinsp_evt **puevt)
 		// mode and the category of this event is internal.
 		if(!(m_isinternal_events_enabled && (cat & EC_INTERNAL)))
 		{
-			*puevt = evt;
```

Comment:
Why were we setting the exit event when returning a FILTERED res?
cc @gnosek

Reply:
I'm not sure tbh, but we don't seem to rely on it anywhere.

Reply:
I think because we wanted to keep parity with the `evt->is_filtered_out` check that we had since always. That way consumers can still receive the event by also knowing it has been filtered out.

Reply:
Ok i think we can retain the old behavior!

Reply:
Done @jasondellaluce .